fjord/analytics/static/css/dashboard.less

```diff
@@ -514,3 +514,7 @@ table.flagged {
     }
   }
 }
+
+.summarytable tbody tr.test-answer {
+   background-color: #fee;
+}
```

Comment:
LESS is hierarchical. This should be moved into the summarytable section of the file.